Sources/Hypertext.swift

```diff
@@ -49,8 +49,9 @@ extension Array: Renderable {
 }
 
 open class tag: Renderable {
-    public var name: String? = nil
-    public var isSelfClosing: Bool = false
+    open var isSelfClosing: Bool { return false }
+    open var name: String { return String(describing: type(of: self)) }
```

Comment:
I know this works, but does this work because `name` is the only `String` property of `tag`?